package.json

```diff
@@ -71,7 +71,9 @@
     "markdown": "^0.5.0",
     "react": "^16.12.0",
     "react-contextmenu": "^2.13.0",
-    "react-dnd-test-backend": "^2.6.0",
+    "react-dnd": "^10.0.2",
+    "react-dnd-html5-backend": "^10.0.2",
```

Comment:
Looks like `react-dnd` and `react-dnd-html5-backend` are `dependencies` now, not `devDependencies`.

Reply:
Yep, I missed that after the addon removal.